script/foreman-debug

```diff
@@ -86,21 +86,24 @@ add_file() {
   MIME=$(file -bi "$FILE" | cut -d\; -f1)
   case $MIME in
         application/x-gzip)
-          zcat "$FILE" | sed -r "$FILTER" > "$DIR$FILE.txt"
-	  touch -c -r "$FILE" "$DIR$FILE.txt"
+          OUTFILE=$(echo "$FILE" | sed s/.gz$//)
```

Comment:
Bash allows you to replace strings in variables, it's built in feature. Why not to use that.

http://stackoverflow.com/questions/13210880/replace-one-substring-for-another-string-in-shell-script